examples/supplychain-app/quorum/smartContracts/compile.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path'); 
+const fs = require('fs');
+const solc = require('solc'); 
+
+function GetByteCode(numberOfIterations){
+  // Importing the contract main entrypoint and the supporting files for the smart contract
+  const general = path.resolve(__dirname,'contracts','General.sol');
```

Comment:
Prefer the input directory to be parameterized i.e. take the dir as input to the js file and compile all the contracts in it.